Content.Server/Atmos/Piping/Unary/Components/GasVentPumpComponent.cs

```diff
@@ -59,6 +59,48 @@ public sealed partial class GasVentPumpComponent : Component
         [DataField("underPressureLockoutLeaking")]
         public float UnderPressureLockoutLeaking = 0.0001f;
 
+        #region some averaging stuff
+        [ViewVariables(VVAccess.ReadOnly)]
+        public int samples { get; set; } = 0;
```

Comment:
Please follow the C# convention of public fields having their first character Capitalized.